### ISSUE TITLE
Fail gracefully when Jupyter notebook requirements are missing

### ISSUE THREAD

**Issue by MridulS** [econ-ark/HARK#719]
- initial import at top, should be able to give beginners better error messages that explain the import errors and installation errors, or point them to a launch URL (econ-ark) or MyBinder.

**Comment by alanlujan91** [comment]
jupyter is not a HARK requirement 

https://github.com/econ-ark/HARK/blob/master/requirements/base.txt

**Comment by mnwhite** [comment]
But then the user wouldn't even *begin* to be able to run a notebook and
encounter missing requirements, right?

**Comment by MridulS** [comment]
Yes we shouldn't have `jupyter` as a requirement. It's a frontend requirement to run the examples, it's not a requirement for HARK to work. Adding `jupyter` as a core requirements will add that to the packaged pip and conda package which adds a ton of bloat when someone does `pip/conda install econ-ark`. This also pollutes the environment.

I think this issue may not be still relevant but we need docs or some command to make sure users can set it up locally.
